### DISCUSSION TITLE
Using the shovel as a base for a mace

### DISCUSSION THREAD

**Discussion by Eldritch6847** [mickelus/tetra#415] [Feature Requests]
A mace would be a good for causing extra damage to armored opponents, while still retaining some form of speed. With Light Handles, throwing maces would be possible. Throwing maces were actually used by the Byzantines. More info here: https://www.reddit.com/r/AskHistorians/comments/238rkc/what_kind_of_mace_would_a_byzantine_cataphract_use/

**Comment by mickelus** [comment]
So it would work like a warhammer?

**Reply by Eldritch6847** [reply]
Something like that, just swings a tad faster and can be thrown if using the light handle
